File: CS/20_Print/SilentlyPrintPPTByDefaultPrinter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using System.Drawing.Printing;

namespace SilentlyPrintPPTByDefaultPrinter
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PowerPoint document.
            Presentation presentation = new Presentation();

            //Load the file from disk.
            presentation.LoadFromFile(@"..\..\..\..\..\..\Data\Template_Ppt_6.pptx");

            //Print the PowerPoint document to default printer.
            PresentationPrintDocument document = new PresentationPrintDocument(presentation);
            document.PrintController = new StandardPrintController();

            presentation.Print(document);
        }
    }
}

File: VB/20_Print/SilentlyPrintPPTByDefaultPrinter.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation
Imports System.Drawing.Printing

Namespace SilentlyPrintPPTByDefaultPrinter
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PowerPoint document.
			Dim presentation As New Presentation()

			'Load the file from disk.
			presentation.LoadFromFile("..\..\..\..\..\..\Data\Template_Ppt_6.pptx")

			'Print the PowerPoint document to default printer.
			Dim document As New PresentationPrintDocument(presentation)
			document.PrintController = New StandardPrintController()

			presentation.Print(document)
		End Sub
	End Class
End Namespace